sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Demo/OTEL_DIAGNOSTICS.json

```diff
@@ -0,0 +1,7 @@
+// OpenTelemetry Self-diagnostics configuration.
```

Comment:
Why is this included?

Reply:
thanks for calling this out. github wasn't showing that, i had to refresh the page.
That wasn't supposed to be checked in. That's from some other tests I was doing earlier in the week.
i removed the file.